### DISCUSSION TITLE
First release?

### DISCUSSION THREAD

**Discussion by drexlerd** [rleap-project/dlplan#22] [General]
Hi,

I think we are ready for our first release version. I am using the library in my latest project where I am using the cpp version in LAPKT and the python version in d2l very easily.

For me, it is very important to have enough tests and examples in the first release. Therefore, I added tests for all elements and improved the existing examples. There is a cpp and a corresponding python example now for both components. Also, I added a guide.pdf in /docs with an overview of all available elements.

Do you think there is anything missing for making the first release?

**Comment by jendrikseipp** [comment]
I like the "release early, release often" approach. And it's great that you have added tests. You might want to start with a 0.1 release instead of 1.0 to indicate that the interface might not be stable yet. Then you can release 1.0 when the first paper is done.
